apps/console/src/features/applications/data/application-templates/templates/saml-web-application/create-wizard-help.tsx

```diff
@@ -50,8 +50,7 @@ const SAMLWebApplicationCreateWizardHelp: FunctionComponent<SAMLWebApplicationCr
                 <Divider/>
                 <Heading as="h5">Protocol</Heading>
                 <p>
-                    The access configuration protocol which will be used to SSO (Single sign-on) to
-                    the application.
+                    The access configuration protocol which will be used to log in to the application using SSO.
```

Comment:
These strings should be coming from the i18n module. Please check.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/1968/commits/708380d96d47fa116fb8d5c83be3489711a9d7de.